packaging/snapd.mk

```diff
@@ -55,14 +55,19 @@ endif
 # The list of go binaries we are expected to build.
 go_binaries = $(addprefix $(builddir)/, snap snapctl snap-seccomp snap-update-ns snap-exec snapd)
 
+GO_TAGS = nosecboot
+ifeq ($(with_testkeys),1)
+GO_TAGS += withtestkeys
```

Comment:
`go help build` says:
```
	-tags tag,list
		a comma-separated list of build tags to consider satisfied during the
		build. For more information about build tags, see the description of
		build constraints in the documentation for the go/build package.
		(Earlier versions of Go used a space-separated list, and that form
		is deprecated but still recognized.)
```
so maybe it's better to do `GO_TAGS="withtestkeys,$GO_TAGS"` (though this is `make` and not `bash`, so it may be that this syntax is not the right one)?

Reply:
Hmm I remember tags were whitespace separated. Looking at release notes, go 1.13 introduced comma separated tags yay! (probably because whitespace separated tags were so hard to get right with shell quoting).